src/main/webapp/app/core/language/language.helper.ts

```diff
@@ -98,7 +98,7 @@ export class JhiLanguageHelper {
         return 'en';
     }
 
-    public getNavigatorReference(): any {
+    public getNavigatorReference(): Navigator {
```

Comment:
The related test is now failing because of this change

Reply:
should be fixed